src/main/kotlin/org/team5419/frc2019offseason/subsystems/Vacuum.kt

```diff
@@ -69,6 +71,10 @@ class Vacuum(
         // else if(!hasPeice && !isPumping){
         //     mTalon.set(ControlMode.PercentOutput, 0.0)
         // }
+        if (!hasPeice && mTalon.getOutputCurrent() >= Constants.Vacuum.RESTING_THRESHOLD) {
+            hasPeice = true
+            mVision.flashForSeconds(0.2)
+        }

```

Comment:
This looks good, might want to consider adding a rolling average

Reply:
what's that?